src/Security/Authentication/OpenIdConnect/src/OpenIdConnectOptions.cs

```diff
@@ -38,6 +40,10 @@ public OpenIdConnectOptions()
         SignedOutCallbackPath = new PathString("/signout-callback-oidc");
         RemoteSignOutPath = new PathString("/signout-oidc");
         SecurityTokenValidator = _defaultHandler;
+        TokenHandler = _defaultTokenHandler;
```

Comment:
Should `TokenHandler` be plural, as in the WsFed and JwtBearer PRs? Or all singular? @keegan-caruso

Reply:
we discussed off-line. resolved.